### PR TITLE
bug fix on number not displaying

### PR DESCRIPTION
**Pull request by knil-maloon**
Quick fix from the number not displaying on the data builder.

### CODE REVIEW

File: sop/sop_step_1_save_AJAX.php [hunk 1]
```diff
@@ -126,6 +126,7 @@
 $data['sop_due_d_preview'] = $date->format('d F Y');
 
 $data['concept_id'] = $concept_id;
+$data['record_id'] = $record;
```
Comment (paulj3) [inline sop/sop_step_1_save_AJAX.php:129]: I have a question for you...in naming this variable, why did you choose `$record` over `$record_id`?

Comment (knil-maloon) [reply sop/sop_step_1_save_AJAX.php:129]: That is a good question that the Eva of 5 years ago might be able to reply, heh. Nowadays I usually add the 'id'.

Comment (paulj3) [reply sop/sop_step_1_save_AJAX.php:129]: @knil-maloon I see ~45 instances of `$record` in this file.  Would you be up for refactoring the variable name?  it's really easy to do in most IDEs.

Comment (paulj3) [reply sop/sop_step_1_save_AJAX.php]: @knil-maloon wonderful! thank you

**Review by paulj3 — APPROVED** [review]

**Pull request merged**